packages/deployment/app/config.yml

```diff
@@ -30,9 +30,6 @@ default:
   processDefaultDeadLetterQueues: true
 
   nested_templates:
-    CumulusApiDistribution:
```

Comment:
With this change it looks like the lambda in the `distribution_api_lambdas.yml` file won’t be deployed. If that’s the case, should this PR also be disabling the distribution/s3access integration tests that make use of that lambda?

That is, `example/spec/parallel/s3Access/s3AccessSpec.js` eventually makes use of https://github.com/nasa/cumulus/blob/master/packages/integration-tests/api/distribution.js#L19 which will fail without the `ApiDistribution` lambda?

Reply:
I left in our tests for that purpose, since I feel like we should still have our tests, but should eventually rip all that out (maybe that should be now since we're not supporting the dist api anymore? feels safer to leave it for now).

The changes here leave the dist API in our deployment for the tests: https://github.com/nasa/cumulus/pull/1082/files#diff-3e433ed2969e4942419d02f57b089cfc

Reply:
Nice :+1: - I’m a fan of leaving things in for now and then maybe eventually making it something we only test in master builds or release builds.